src/wallet/mod.rs

```diff
@@ -749,13 +749,22 @@ where
             params.bumping_fee.is_some(), // we mandate confirmed transactions if we're bumping the fee
         )?;
 
+        let weighted_drain_txout = WeightedTxOut {
```

Comment:
This needs to be updated? Do you need help figuring out how to do it? :)

Reply:
No, or yes :) Changes with this may collide with the solution for #147 . I don't know if I should address that in this same PR or not.

Reply:
I think you should fork from bdk master and try to implement #147 first (you can of course go back and look at this code if you feel like there are problems that you solved here already). In this way you'll come up with a solution that doesn't depend on the waste metric.